docs/01-start-here/01-installation-steps.md

```diff
@@ -23,27 +23,49 @@ Follow [this link](https://www.google.co.uk) and enter the relevant search strin
 3. Submit a PR
 
 # Local Test Server setup
+
+## Docker
+
+1. Install Docker 
+    - (See [documentation](https://docs.docker.com/engine/installation/))
+2. Clone repository
+    - `git clone git@github.com:guardian/frontend.git && cd frontend`
+3. Get [janus credentials](https://janus.gutools.co.uk/) for frontend and CMS fronts 
+4. Get properties file
+    - `aws s3 cp --profile frontend s3://aws-frontend-store/template-frontend.properties "~/.gu/frontend.properties"`
+5. Build the container image
+    - `docker-compose build dev`
+6. Run the container (dev service)
+    - `docker-compose run --service-ports dev`
+7. Build and run the app within the container
+    - `make build compile && ./sbt`
```

Comment:
I can see the appeal of non-interactive run commands in general, but here I think an interactive sbt prompt is way better than a forced `dev-build run`. We need to be able to use the sbt prompt for switching projects, running tests, etc.